src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -15,6 +15,9 @@ public class HttpProtocolTests : HttpClientTest
         protected virtual Stream GetStream(Stream s) => s;
 
         [Theory]
+        // The following disabled by ActiveIssue: 26540
+        // [InlineData("HTTP/1.1 200      ", 200, "     ")]
+        // [InlineData("HTTP/1.1 200      Something", 200, "     Something")]
```

Comment:
We should mark the issue with disabled-test label once you check in.

Reply:
Ok.